voronoi.cpp

```diff
@@ -106,16 +119,21 @@ multi_polygon_type_fp Voronoi::build_voronoi(
                 current_edge = current_edge->rot_next();
             }
         } while (current_edge != start_edge);
-        ring.push_back(ring.front());  // Close the ring.
-        size_t poly_index = std::distance(segments_to_poly.cbegin(), std::upper_bound(segments_to_poly.cbegin(), segments_to_poly.cend(), edge.twin()->cell()->source_index()));
-        if (bg::area(ring) > 0) {
-            // This is the outer ring of the poly.
-            output[poly_index].outer().swap(ring);
-        } else {
-            // This has negative area, it must be a hole in the outer.
-            output[poly_index].inners().push_back(ring);
+
+        if(!ring.empty())
```

Comment:
I can see how this will solve a problem but if that ring is ever empty, it means that the Voronoi routine did something unexpected.  The output from pcb2gcode might be garbage in that case.  I don't have armhf available so I can't test.  If you can, look at an armhf output and check if it looks reasonable?

Obviously there is no effect on the already-not-crashing unit tests and integration tests, confirmed by TravisCI, so the code at least not going to break anything!